src/Icons/Icons.tsx

```diff
@@ -92,6 +92,9 @@ import filtersUrl, { ReactComponent as Filters } from './icons/filters.svg';
 import standardArrowUrl, { ReactComponent as StandardArrow } from './icons/standard-arrow.svg';
 import bringToFrontUrl, { ReactComponent as BringToFront } from './icons/bring-to-front.svg';
 import sendToBackUrl, { ReactComponent as SendToBack } from './icons/send-to-back.svg';
+import alignLeftUrl, { ReactComponent as AlignLeft } from './icons/left-align.svg';
+import alignCenterUrl, { ReactComponent as AlignCenter } from './icons/center-align.svg';
+import alignrightUrl, { ReactComponent as Alignright } from './icons/right-align.svg';
 
 const DEFAULT_ICON_COLOR = '#4c4c4c';
```

Comment:
You should be able to change this colour and all the icons in Storybook will change colour (with the exception of the app switcher icon). You can make sure this happens by removing the fill from all elements of the svg except the svg element itself.

Reply:
fixed